models/index.js

```diff
@@ -20,11 +21,14 @@ Rent.belongsTo(Truck);
 Rent.hasOne(Review);
 Review.belongsTo(Rent);
 
+Feature.hasOne(Truck);
+Truck.hasOne(Feature);
```

Comment:
A truck has many features

Reply:
And A feature belongs to a truck (hasOne does the opposite of what its name implies)

Reply:
Ah. I understand now.